server/src/main/scala/controllers/application/WithExports.scala

```diff
@@ -27,6 +27,8 @@ import scala.util.Try
 trait WithExports {
   self: Application =>
 
+  val bestPax: Arrival => Int = ArrivalHelper.bestPax
```

Comment:
I'm wondering if there might be a single place we could set this function. From the PR it's mentioned 5 times so has the possibility of getting out of sync if a new function replaces it but misses some of the usages

Reply:
I was using different versions of it previously, but pulled those out, so will see where it's not referencing the original

Reply:
Possibly fine though as it's fairly obviously a centrally used function. We don't want to have to pass it through loads of calls if that would be necessary

Reply:
yeah, most of it was totally unnecessary - I've pulled out the extra references.